app-emulation/lxd/lxd-0.18.ebuild

```diff
@@ -0,0 +1,172 @@
+	test? (
+		app-misc/jq
+		dev-db/sqlite
+		net-misc/curl
+		sys-devel/gettext
+	)
+"
+
+RDEPEND="
+	daemon? (
+		app-admin/cgmanager
+		app-arch/xz-utils
+		app-emulation/lxc[cgmanager]
+		net-analyzer/openbsd-netcat
+		net-misc/bridge-utils
+		net-misc/rsync[xattr]
+		sys-apps/iproute2
+		virtual/acl
+		btrfs? (
+			sys-fs/btrfs-progs
+		)
+		criu? (
+			sys-process/criu
+		)
+		lvm? (
+			sys-fs/lvm2
+		)
+	)
+"
+
+# KNOWN ISSUES:
+# - Translations may not work.  I've been unsuccessful in forcing
+#   localized output.  Anyway, upstream (Canonical) doesn't install the
+#   message files.
+
+# TODO:
+# - since 0.15 gccgo is a supported compiler ('make gccgo').  It would
+#   be preferable for that support to go into the golang-build eclass not
+#   this package directly.
+# - Add apparmor USE.  There are some exec calls to apparmor_parser
+# - Test build with Go 1.4 & 1.5
+# - Add seccomp USE - upstream release notes for 0.18 mention it, a global
+#   USE flag exists
+# - check daemon flags, need --force-local in conf.d?
+# - where is --force-local (0.18)? not documented in --help
+# - integrate "lxd shutdown" into initscript as custom action (default "stop"
+#   action should _not_ stop containers amirite?)
+#   "Perform a clean shutdown of LXD and all running containers"
+
+src_prepare() {
+	cd "${S}/src/${EGO_PN}"
+
+	epatch "${FILESDIR}/${P}-dont-go-get.patch"
+
+	if use daemon; then
+		# Upstream requires the openbsd flavor of netcat (with -U), but
+		# Gentoo installs that with a renamed binary
+		epatch "${FILESDIR}/${P}-nc-binary-name.patch"
+	fi
+
+	# Warn on unhandled locale changes
+	l10n_find_plocales_changes po "" .po
+}
+
+src_compile() {
+	golang-build_src_compile
+
+	cd "${S}/src/${EGO_PN}"
+
+	if use daemon; then
+		# Build binaries
+		GOPATH="${S}:$(get_golibdir_gopath)" emake
+	else
+		# build client tool
+		GOPATH="${S}:$(get_golibdir_gopath)" emake client
+	fi
+
+	use nls && emake build-mo
+}
+
+src_test() {
+	if use daemon; then
+		# Go native tests should succeed
+		golang-build_src_test
+	fi
+}
+
+src_install() {
+	# Installs all src,pkg to /usr/lib/go-gentoo
+	golang-build_src_install
+
+	cd "${S}"
+	dobin bin/lxc
+	if use daemon; then
+		dobin bin/fuidshift
+
+		dosbin bin/lxd
+	fi
+
+	cd "src/${EGO_PN}"
+
+	if use nls; then
+		for lingua in ${PLOCALES}; do
+			if use linguas_${lingua}; then
+				domo po/${lingua}.mo
+			fi
+		done
+	fi
+
+	if use daemon; then
+		newinitd "${FILESDIR}"/${P}.initd lxd
+		newconfd "${FILESDIR}"/${P}.confd lxd
+
+		systemd_dounit "${FILESDIR}"/lxd.service
+	fi
+
+	newbashcomp config/bash/lxc.in lxc
+
+	dodoc AUTHORS CONTRIBUTING.md README.md
+
+	docinto specs
+	dodoc specs/*
+}
+
+pkg_postinst() {
+	einfo
+	einfo "Consult https://wiki.gentoo.org/wiki/LXD for more information,"
+	einfo "including a Quick Start."
+
+	# The messaging below only applies to daemon installs
+	use daemon || return 0
+
+	# The control socket will be owned by (and writeable by) this group.
+	enewgroup lxd
```

Comment:
What if I drop the daemon USE flag?  This group will be orphaned on the system and should it?